db-service/lib/common/DatabaseService.js

```diff
@@ -6,6 +6,12 @@ const cds = require('@sap/cds/lib')
 /** @typedef {unknown} DatabaseDriver */
 
 class DatabaseService extends cds.Service {
+
+  init() {
+    cds.on('shutdown', () => this.disconnect())
```

Comment:
We might need to implement something that disconnects all tenant pools in multitenancy mode. Something like `disconnect('*')`

Reply:
`disconnect(undefined)` should match all tenants. Will update the `disconnect` function to just go over all pools.